src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -11,16 +12,23 @@ Cypress.Commands.add('fillVaTextInput', (field, value) => {
     element
       .shadow()
       .find('input')
-      .as('currentElement')
-      .clear({ force: true })
-      .type(strValue, { force: true });
+      .as('currentElement');
+
+    cy.get('@currentElement').click();
+
+    cy.get('@currentElement').clear(DELAY_OPTION);
+
+    if (strValue !== '') {
+      // using type requires a non-empty value
+      cy.get('@currentElement').type(strValue, FORCE_OPTION);
+    }
```

Comment:
Explanation: we kept on having failures where cypress would report that it couldn't type into a disabled field, even though the field was clearly not disabled. Adding the .click to this section resolved the issue to make sure the right input was active.